src/playsim/mapthinkers/a_lightning.cpp

```diff
@@ -48,11 +48,12 @@ IMPLEMENT_CLASS(DLightningThinker, false, false)
 //
 //----------------------------------------------------------------------------
 
-void DLightningThinker::Construct()
```

Comment:
I would highly recommend doing this instead (note that I don't know if the syntax is EXACTLY right, but you get the idea :P)
`inline void DLightningThinker::Construct() : DLightningThinkerConstruct("world/thunder"){}`